astroquery/atomic/core.py

```diff
@@ -271,46 +272,44 @@ def _parse_result(self, response):
                     # maintain table dimensions when data missing
                     row.append('None')
             if row:
-                input.append('\t'.join(row))
-        if input:
-            return ascii.read(input, data_start=0, delimiter='\t',
+                result_data.append('\t'.join(row))
+        if result_data:
+            return ascii.read(result_data, data_start=0, delimiter='\t',
                               names=colnames, fast_reader=False)
         else:
             # return an empty table if the query yielded no results
             return Table()
 
-    def _submit_form(self, input=None, cache=True):
+    def _submit_form(self, input_data=None, cache=True):
         """Fill out the form of the SkyView site and submit it with the
-        values given in `input` (a dictionary where the keys are the form
+        values given in ``input_data`` (a dictionary where the keys are the form
         element's names and the values are their respective values).
 
         """
-        if input is None:
-            input = {}
-        # only overwrite payload's values if the `input` value is not None
+        if input_data is None:
+            input_data = {}
+        # only overwrite payload's values if the ``input_data`` value is not None
         # to avoid overwriting of the form's default values
-        payload = self._default_form_values.copy()
-        for k, v in input.items():
+        payload = self.__default_form_values.copy()
+        for k, v in input_data.items():
             if v is not None:
                 payload[k] = v
-        url = urlparse.urljoin(self.FORM_URL, self._default_form.get('action'))
+        url = self._form_action_url
         log.debug(f"final payload = {payload} from url={url}")
         response = self._request("POST", url=url, data=payload,
                                  timeout=self.TIMEOUT, cache=cache)
         log.debug("Retrieved data from POST request")
         return response
 
-    @property
     def _default_form_values(self):
-
-        if self.__default_form_values is None:
-            response = self._request("GET", url=self.FORM_URL, data={},
-                                     timeout=self.TIMEOUT, cache=True)
-            bs = BeautifulSoup(response.text, 'html5')
-            self._default_form = form = bs.find('form')
-            self.__default_form_values = self._get_default_form_values(form)
-
-        return self.__default_form_values
+        response = self._request("GET", url=self.FORM_URL, params={},
```

Comment:
you changed `data={}` to `params={}` here... I think either way, we could just exclude this?  It's a very minor thing but I'm not sure what purpose it serves.

Reply:
This was indeed a red herring for the problem but I kept it as it's still a fix even as it's an empty dict so it shouldn't matter in practice. (afaik `data` doesn't have any effect for a GET query, it takes a `params` dict (while POST can have either or both).)